<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PumpingTest</class>
 <widget class="QWidget" name="PumpingTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1624</width>
    <height>796</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="gb_project_informataion">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>280</width>
         <height>160</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>320</width>
         <height>160</height>
        </size>
       </property>
       <property name="title">
        <string>Project Information</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="lbl_project_name">
          <property name="text">
           <string>Project Name:</string>
          </property>
          <property name="buddy">
           <cstring>l_edit_project_name</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="l_edit_project_name"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lbl_project_number">
          <property name="text">
           <string>Project Number:</string>
          </property>
          <property name="buddy">
           <cstring>l_edit_project_number</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="l_edit_project_number"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lbl_client">
          <property name="text">
           <string>Client:</string>
          </property>
          <property name="buddy">
           <cstring>l_edit__client</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="l_edit__client"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lbl_location">
          <property name="text">
           <string>Location:</string>
          </property>
          <property name="buddy">
           <cstring>l_edit_location</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="l_edit_location"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gb_pumping_test">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>160</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>320</width>
         <height>160</height>
        </size>
       </property>
       <property name="title">
        <string>Pumping Test</string>
       </property>
       <layout class="QFormLayout" name="formLayout_10">
        <item row="0" column="0">
         <widget class="QLabel" name="lbl_pumping_test_name">
          <property name="text">
           <string>Name:</string>
          </property>
          <property name="buddy">
           <cstring>l_edit_pumping_test_name</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="lbl_pumping_test_date_time">
          <property name="text">
           <string>Date/Time:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QTimeEdit" name="time_puming_test"/>
          </item>
          <item>
           <widget class="QDateEdit" name="date_pumping_test"/>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lbl_pumping_test_performed_by">
          <property name="text">
           <string>Performed by:</string>
          </property>
          <property name="buddy">
           <cstring>l_edit_pumping_test_performed_by</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="l_edit_pumping_test_performed_by"/>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="l_edit_pumping_test_name"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gb_units">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>160</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>160</height>
        </size>
       </property>
       <property name="title">
        <string>Units</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="2">
         <widget class="QLabel" name="lbl_dimensions_2">
          <property name="text">
           <string>Dimensions:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QComboBox" name="cb_dimensions_2">
          <item>
           <property name="text">
            <string>m</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QComboBox" name="cb_discharge_2">
          <item>
           <property name="text">
            <string>m³/day</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="lbl_discharge_2">
          <property name="text">
           <string>Discharge:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cb_transmissivity_2">
          <item>
           <property name="text">
            <string>m²/day</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lbl_time_2">
          <property name="text">
           <string>Time:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cb_time_2">
          <item>
           <property name="text">
            <string>min</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lbl_transmissivity_2">
          <property name="text">
           <string>Transmissivity:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gb_aquifer_properties">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>280</width>
         <height>160</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>280</width>
         <height>160</height>
        </size>
       </property>
       <property name="title">
        <string>Aquifer Properties</string>
       </property>
       <layout class="QFormLayout" name="formLayout_11">
        <item row="0" column="0">
         <widget class="QLabel" name="lbl_aquifer_thickness">
          <property name="text">
           <string>Thickness[m]:</string>
          </property>
          <property name="buddy">
           <cstring>l_edit_aquifer_thickness</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="l_edit_aquifer_thickness"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lbl_aquifer_type">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Type:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cb_aquifer_type">
          <item>
           <property name="text">
            <string>Unknown</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Confined</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Unconfined</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Leaky</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Fracture</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="table_well_information">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>920</width>
       <height>160</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>920</width>
       <height>160</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btn_create_new_well">
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>320</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Create a New Well</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_submit">
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>320</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lbl_image">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="icon/icon.qrc">:/menu_bar_icon/well_demo.png</pixmap>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icon/icon.qrc"/>
 </resources>
 <connections/>
</ui>
